docs/releases.md

```diff
@@ -0,0 +1,8 @@
+# Releases
+
+## Releases and Versioning
+This repository was created for the development of AWS distribution of Kubeflow as described in [Kubeflow distribution](https://github.com/kubeflow/community/blob/master/proposals/kubeflow-distributions.md) guideline. Although the distribution manifests are hosted in this repository, many of the overlays and configuration files in this repository have a dependency on the manifests published by Kubeflow community in [`kubeflow/manifests`](https://github.com/kubeflow/manifests) repository. Hence, AWS distribution for Kubeflow-v1.3.1 was developed on a fork of the `v1.3-branch` branch from `kubeflow/manifests` repository but it presented several challenges for ongoing maintainence as described in [#76](https://github.com/awslabs/kubeflow-manifests/issues/76). Starting Kubeflow-v1.4, this repository only contains the delta from the released manifests in the `kubeflow/manifests` repository and additional components required for AWS distribution.
+
+Since each AWS distribution release is built on top of a Kubeflow release, the releases follow the following format: `{KUBEFLOW_RELEASE_VERSION}-aws-b{BUILD_NUMBER}` where `KUBEFLOW_RELEASE_VERSION` refers to the [Kubeflow's released version](https://github.com/kubeflow/manifests/releases) and `BUILD_NUMBER` refers to AWS build for that Kubeflow version. `BUILD_NUMBER` uses [semantic versioning](https://semver.org/) (SemVer) to indicate whether changes included in a particular release introduce features or bug fixes and whether or not features break backwards compatibility.
+
+When a Kubeflow on AWS version is released, a Git tag containing a `{KUBEFLOW_RELEASE_VERSION}-aws-b{X.Y.Z}` version is created on the repository. These releases can be found under the repository's [releases](https://github.com/awslabs/kubeflow-manifests/releases) section.
```

Comment:
Any reason we refer to it as `b{X.Y.Z} `but previously we had `b{BUILD_NUMBER}`
This is in reference to the format of `{KUBEFLOW_RELEASE_VERSION}-aws-b{X.Y.Z}` vs `{KUBEFLOW_RELEASE_VERSION}-aws-b{BUILD_NUMBER}` which I believe refer to the same thing?

Reply:
good catch